plugins/PostProcessingPlugin/scripts/ColorMix.py

```diff
@@ -0,0 +1,190 @@
+# ColorMix script - 2-1 extruder color mix and blending
+# This script is specific for the Geeetech A10M dual extruder but should work with other Marlin printers. 
+# It runs with the PostProcessingPlugin which is released under the terms of the AGPLv3 or higher.
+# This script is licensed under the Creative Commons - Attribution - Share Alike (CC BY-SA) terms
+
+#Authors of the 2-1 ColorMix plug-in / script:
+# Written by John Hryb - john.hryb.4@gmail.com
+
+##history / change-log:
+##V1.0.0
+
+## Uses -
+## M163 - Set Mix Factor 
+## M164 - Save Mix - saves to T3 as a unique mix
+
+import re #To perform the search and replace. 
+from ..Script import Script
+
+class ColorMix(Script):
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name":"ColorMix 2-1",
+            "key":"ColorMix 2-1",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "a_unitsOfMeasurement":
```

Comment:
I would rather use our standard format for the settings key. Something like...
```suggestion
                "measurement_units":
```